tests/Fixer/StringNotation/StringLineEndingFixerTest.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Tests\Fixer\StringNotation;
+
+use PhpCsFixer\Tests\Test\AbstractFixerTestCase;
+use PhpCsFixer\WhitespacesFixerConfig;
+
+/**
+ * @author Ilija Tovilo <ilija.tovilo@me.com>
+ *
+ * @internal
+ *
+ * @covers \PhpCsFixer\Fixer\StringNotation\StringLineEndingFixer
+ */
+final class StringLineEndingFixerTest extends AbstractFixerTestCase
+{
+    /**
+     * @param string      $expected
+     * @param null|string $input
+     *
+     * @dataProvider provideFixCases
+     */
+    public function testFix($expected, $input = null)
+    {
+        $this->doTest($expected, $input);
+    }
+
+    public function provideFixCases()
+    {
+        $template = "<?php\n\$a=\n<<<'EOT'\n%s\n\nEOT;\n";
```

Comment:
Please add similar test cases with heredoc (`<<<EOT`) instead of nowdoc (`<<<'EOT'`).

Reply:
@iluuu1994 or @julienfalque could one of you add that one missing test?

@julienfalque , if I get you right, this PR is RTM from your perspective after that test is added ? So we would merge it

Reply:
I added the test cases, we can merge the PR now I think :)

Please squash my commit when merging.